index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-array-constructor */
+/* eslint-disable semi */
+const bookTitle = document.getElementById('book-title');
+const authorName = document.getElementById('book-author');
+const button = document.getElementById('submit');
+const list = document.querySelector('#books-ul');
+const array = new Array();
+let counter = 0;
+
+// eslint-disable-next-line no-unused-vars
+const removeBook = (e) => {
+  const remotion = e.target.parentNode;
+  list.removeChild(remotion);
+}
+
+const dataform = () => localStorage.setItem('data', array);
```

Comment:
- [ ] [REQUIRED] Please read this article about [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md) because your local storage is not working in my case. 

- Data should preserved in the browser's memory by using localStorage, reload the page manually to check this after adding a book and removing a book.